frontend/packages/kubevirt-plugin/integration-tests/tests/non-admin.scenario.ts

```diff
@@ -29,53 +44,119 @@ const {
 } = process.env;
 
 describe('Kubevirt non-admin Flow', () => {
-  const leakedResources = new Set<string>();
-  const vm = new VMBuilder(getBasicVMBuilder())
-    .setNamespace(testNonAdminNamespace)
-    .setProvisionSource(ProvisionSource.URL)
-    .setDisks([rootDisk])
-    .build();
+  const pvc = new PVC(RHEL7_PVC);
 
   beforeAll(async () => {
+    // Prepare golden pvc for normal user.
+    execSync(`curl --fail ${REMOTE_CIRROS_IMAGE} -o ${LOCAL_CIRROS_IMAGE} > /dev/null`);
+    await pvc.create();
+
     await loginView.logout();
     await loginView.login(BRIDGE_HTPASSWD_IDP, BRIDGE_HTPASSWD_USERNAME, BRIDGE_HTPASSWD_PASSWORD);
     await createProject(testNonAdminNamespace);
-    await browser.get(`${appHost}/k8s/ns/${testNonAdminNamespace}/virtualization`);
-    await isLoaded();
   });
 
   afterAll(async () => {
-    removeLeakedResources(leakedResources);
+    execSync(`rm ${LOCAL_CIRROS_IMAGE}`);
     execSync(`kubectl delete --ignore-not-found=true project ${testNonAdminNamespace}`);
     await loginView.logout();
     await loginView.login(KUBEADMIN_IDP, KUBEADMIN_USERNAME, BRIDGE_KUBEADMIN_PASSWORD);
+    await pvc.delete();
```

Comment:
IMO we should cleanup the resources with kubectl and not the UI

Reply:
The problem is deleting pvc via kubectl does not work, it will be re-created. Unless delete the datavolume.
Delete pvc from the UI simply works well.

Reply:
let's delete the DV